.github/workflows/i18n-pull.yml

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 Graz University of Technology.
+#
+# Invenio-Users-Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+name: i18n:pull translations
```

Comment:
For this ones, you might want to contact Lars so he can put a template in the organization. That way you can simply select + add (saves time).

Reply:
that's a good idea - I don't mind - i wanted to test and push the new command anyway.
this is the last one. for this template.. I think :)
The rest needs different templates since they are combined `JS` and ``python` translations.
Thanks!